node/internal.ts

```diff
@@ -727,7 +733,9 @@ export function _loadData(): void {
                 ++loaded;
                 _debug('loading ' + envvar);
                 _vault.storeSecret(envvar, value);
-                delete process.env[envvar];
+                if (!_startsWith(envvar, 'VSTS_TASKVARIABLE_')) { // avoid deleting public variables
```

Comment:
Does it mean that avoiding removal of these variables would expose them for further tasks?

Reply:
@anatolybolshakov Good point. I changed the code to continue deleting it to avoid exposing them for further tasks. Can you check it again?